lms/djangoapps/experiments/views_custom.py

```diff
@@ -160,13 +166,13 @@ def get(self, request):
             return Response({
                 'show_upsell': show_upsell,
                 'price': upgrade_price,
-                'basket_url': basket_link,
+                'basket_url': basket_url,
             })
         else:
             return Response({
                 'show_upsell': show_upsell,
                 'upsell_flag': MOBILE_UPSELL_FLAG.is_enabled(),
                 'experiment_bucket': bucket,
                 'user_upsell': user_upsell,
-                'basket_link': basket_link,
```

Comment:
If you haven't already, let the mobile team know about this -- we want to make sure that they are not expecting basket_link.

Reply:
Yes, I'll make sure the expectation is set that if `show_upsell` is false, all other values should be ignored except as debugging aids.